cmd/rc/rc.go

```diff
@@ -155,7 +155,11 @@ func ParseOptions(options []string) (opt map[string]string) {
 // If the user set flagName set the output to its value
 func setAlternateFlag(flagName string, output *string) {
 	if rcFlag := pflag.Lookup(flagName); rcFlag != nil && rcFlag.Changed {
-		*output = rcFlag.Value.String()
+		if sliceValue, ok := rcFlag.Value.(pflag.SliceValue); ok {
+			*output = sliceValue.GetSlice()[0]
```

Comment:
Is there any way that `sliceValue.GetSlice()` can return 0 elements? If so then this needs a bit of `if len(...) > 0 {`

Reply:
I wouldn't think this since we are already checking whether the flag has been changed above. I will add a check anyway.

Reply:
Should be fixed.